bpf/bpf_host.c

```diff
@@ -450,20 +454,25 @@ resolve_srcid_ipv4(struct __ctx_buff *ctx, __u32 srcid_from_proxy,
 	return src_id;
 }
 
+struct {
+	__uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+	__type(key, __u32);
+	__type(value, struct ct_buffer4);
+	__uint(max_entries, 1);
+} CT_TAIL_CALL_BUFFER4 __section_maps_btf;
+
```

Comment:
For my own education (I'll need to add a per-CPU map for some other use soon) - is there an obvious advantage to having separate IPv4/IPv6 maps? I would have instinctly gone with a single map, containing a union of the `ct_buffer4` / `ct_buffer6` structs.